BackofficeBundle/Form/Type/DatePickerType.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace OpenOrchestra\BackofficeBundle\Form\Type;
+
+use Symfony\Component\Form\Extension\Core\Type\DateType;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\Form\FormView;
+
+/**
+ * Class DatePickerType
+ */
+class DatePickerType extends DateType
+{
+
+    /**
+     * @param FormView $view
+     * @param FormInterface $form
+     * @param array $options
+     */
+    public function finishView(FormView $view, FormInterface $form, array $options)
+    {
+        $view->vars['widget'] = $options['widget'];
+        // Change the input to a HTML5 date input if
+        //  * the widget is set to "single_text"
+        //  * the format matches the one expected by HTML5
+        //  * the html5 is set to true
+        if ($options['html5'] && 'single_text' === $options['widget'] && self::HTML5_FORMAT === $options['format']) {
+            $view->vars['type'] = 'date';
+        }
+
+        if($options['widget'] === 'single_text') {
```

Comment:
the datePicker work only with the single_text option so why no force this options. 
If an user can user an other he uses the DateType and no DatePickerType